Client/Frontend/Browser/BrowserViewController/BrowserViewController+WKNavigationDelegate.swift

```diff
@@ -135,6 +139,66 @@ extension BrowserViewController: WKNavigationDelegate {
             decisionHandler(.cancel)
             return
         }
+        
+        if isUpholdOAuthAuthorization(url) {
+            decisionHandler(.cancel)
+            guard let queryItems = URLComponents(url: url, resolvingAgainstBaseURL: false)?.queryItems else {
+                return
+            }
+            var items: [String: String] = [:]
+            for query in queryItems {
```

Comment:
Can change this to (if you want):

`let items = queryItems.flatMap({ [$0.name : $0.value] })`

Reply:
Ah, that was the solution I was looking for, I tried a `reduce` before just doing this lol. To be honest though, I'll probably stick with current way as its easier to read